### PR TITLE
Bootstrap application

### PR DESCRIPTION
**Pull request by jackjohn7**
Bootstrap application

### CODE REVIEW

**Comment by jackjohn7** [comment]
[Documentation changes](https://github.com/Channel-Surfers/Channel-Surfers/wiki)

**Comment by jackjohn7** [comment]
I'm unable to run the e2e tests on my machine since it's arch-based. Playwright isn't playing nice. Will have to wait for CI to find out if the test works.

**Comment by funnyboy-roks** [comment]
Can you add the last-run to the gitignore?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+
+To create a production version of your app:
+
+```bash
+npm run build
+```
+
+You can preview the production build with `npm run preview`.
+
+> To deploy your app, you may need to install an [adapter](https://kit.svelte.dev/docs/adapters) for your target environment.
```
Comment (funnyboy-roks) [inline README.md:38]: We don't really need all of this noise, just the 
```suggestion
## Developing

```sh
npm i -D # Install dependencies
npm run dev # Start server
```
```

**Review by funnyboy-roks — CHANGES_REQUESTED** [review]
Looks good, just a note on the readme

File: README.md (hunk 1 above)
Comment (jackjohn7) [reply README.md]: Agreed. I removed all aside from *Developing* and *Building*

**Review by funnyboy-roks — APPROVED** [review]

**Pull request merged**